app/src/main/java/com/example/byitscover/ReviewPage.java

```diff
@@ -326,15 +329,15 @@ private void setGoogleBooksInfo(View view, BookListing listing) {
      * @param listing is the listing from Storygraph
      */
     private void setStorygraphInfo(View view, BookListing listing) {
-        //set Google rating
+        //set storygraph rating
         TextView storygraphResultRating = (TextView) view.findViewById(R.id.storygraphRating);
         try {
             storygraphResultRating.setText(listing.getAggregateRating().toString());
         }
         catch (Exception e) {
             System.out.println(e.toString());
```

Comment:
Add `storygraphResultPrice.setText("Pricing Info Not Provided");`?

Reply:
Yeah, can do just in case something really weird happens